backend/db/ddl-11-all_csets.jinja.sql

```diff
@@ -1,6 +1,7 @@
 -- Table: all_csets ----------------------------------------------------------------------------------------------------
 DROP TABLE IF EXISTS {{schema}}all_csets{{optional_suffix}} CASCADE;
 
+DROP TABLE IF EXISTS {{schema}}cset_term_usage_rec_counts;
```

Comment:
## Bugfix: `relation "cset_term_usage_rec_counts" already exists`
I think this happened because table was created and process exited before it could be deleted. So I added this line right before table creation as a fallback.